modules/tiles/src/tileset/tileset-3d.ts

```diff
@@ -97,6 +98,7 @@ type Props = {
   token: string;
   attributions: string[];
   headers: any;
+  fetch: any;
```

Comment:
Why do we need a separate prop for fetch? Why can't we just ask the user to put it in the standard `loadOptions` object. This is also documented in deck.gl https://deck.gl/docs/developer-guide/loading-data

Reply:
https://github.com/visgl/deck.gl/blob/f73ab0550231cd9b3fd6ec7363512fa38d178cd8/modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js#L120

Reply:
I don't understand the problem in that line

Reply:
Done